front/src/applications/operationalStudies/views/Project.tsx

```diff
@@ -84,7 +83,7 @@ export default function Project() {
   ];
 
   const { data: project, isError: isProjectError } = osrdEditoastApi.useGetProjectsByProjectIdQuery(
-    { projectId: projectId as number },
+    { projectId: Number(projectId) },
```

Comment:
Then you need to check everywhere that projectId is not invalid
```suggestion
    { projectId },
    {
      skip: projectId === INVALID_ID,
    }
```

Also, you need to update the useEffect which redirect the user if the id is invalid. You can test this case by setting the id to 'a' for instance
```
  useEffect(() => {
    if (projectId === INVALID_ID) {
      navigate('/operational-studies/projects');
    }
  }, [projectId]);
```